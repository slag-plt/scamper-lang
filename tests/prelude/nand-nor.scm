(nand #t #f #t #f #t #f)
(nand #f #f #f)
(nand #t #t #t)
(nand #t #f)
(nand #t #f #f #f)
(nor #t #f #t #f #t #f)
(nor #f #f #f)
(nor #t #t #t)
(nor #t #f)
(nor #t #f #f #f)