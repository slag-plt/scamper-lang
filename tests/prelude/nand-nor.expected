#t
#t
#f
#t
#t
#f
#t
#f
#f
#f
